tailcfg/tailcfg.go

```diff
@@ -1698,10 +1699,17 @@ var FilterAllowAll = []FilterRule{
 	},
 }
 
+// DNSResolver embeds dnstype.Resolver and stores
+// additional configuration.
```

Comment:
This says what it is (which can be seen below), but not really what it's used for. Maybe something like
```
// DNSResolver describes a single DNS resolver and any special handling needed when
// using that resolver.
```

I guess that's not significantly more informative but I feel like it reads better?